documentation/user_docs/docs/manual/Cutting_data_of_interest_from_SQW_files_and_objects.rst

```diff
@@ -1009,9 +1009,9 @@ limited and mostly discouraged.
 - ``cut_sqw`` is fully equivalent to ``cut`` except that attempting to apply it
   to a ``dnd`` object or file, will raise an error.
 
-- ``cut_dnd`` is equivalent to ``cut`` except it only ever returns a ``dnd`` as
-  though ``-nopix`` had been passed.
-
+- ``cut_dnd`` is equivalent to ``cut`` on a ``dnd`` object, but for an
+  ``sqw`` object, it operates on the ``dnd`` (binned image) of the
+  ``sqw`` rather than the pixel information.
```

Comment:
meaning that the pixels are discarded? If not, and the pixels are not cut, image and pixels are not in sync

Reply:
It discards the pixels information in the sense that it ignores that it ever existed. I.e. operates on the `dnd`.